File: HoYa.Web/src/@entities/attribute.ts
import { Definition } from "./entity"; 
//實例化Recipe
export class Attribute extends Definition {
    level: number;
    valueType: string;
    valueNumber: number;
    itemIds: string;
    inventoryIds: string;
    categoryIds: string;
    _checked: boolean;
    constructor() {
        super();
        this.level = 1;
        this.valueType = "";
        this.valueNumber = 1;
        this.itemIds = "";
        this.inventoryIds = "";
        this.categoryIds = "";
        this.value = "";
        this.code = "";
    }
}

File: HoYa/Entities/Attribute.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace HoYa.Entities
{
    public class Attribute : Definition
    {
        public string Value { get; set; }
        public int Level { get; set; }
        public string ValueType { get; set; }
        public int ValueNumber { get; set; }
        public string ItemIds { get; set; }
        public string InventoryIds { get; set; }
        public string CategoryIds { get; set; }
    }

    public class AttributeGroup : Relation<Attribute, Inventory>
    {
    }
}